### ISSUE TITLE
Blank white page for any user without a linked Employee record

### ISSUE THREAD

**Issue by wreckage0907** [rtCamp/next-pms#2076]
# 🐛 Bug Report

## **Description**
We used to display that the user doesnt have an employee linked with them to use the app but currently it shows a blank white page 

## **Steps to Reproduce**
1. Go to <SITE>/next-pms logged in as administrator 
2. observe blank white page

## **Expected Behavior**
- an error page explaining why they are not allowed 

## **Actual Behavior**
- blank page 

## **Screenshots/Screencasts**

<img width="1920" height="960" alt="Image" src="https://github.com/user-attachments/assets/2d6e80f3-140e-4846-ac94-1b91b7f3ce47" />

## **Environment**
- OS: [e.g., Windows 10, macOS 13]
- Browser: [e.g., Chrome 109, Firefox 90]

## **Additional Context**
[Add any other context about the problem here.]

**Comment by renishsurani** [comment]
I have verified that a user without a linked employee is getting the proper error page when trying to access the `/timesheet` page or any other frontend page.

<img width="2059" height="1149" alt="Image" src="https://github.com/user-attachments/assets/bd0facff-f599-4de9-a73d-cf64b7fd69b1" />

cc: @Ajay-Jaiswal416

**Comment by renishsurani** [comment]
We need to check by logging in as an administrator. (As the admin does not have an employee created)

cc: @Ajay-Jaiswal416
